src/alliteratedActors.js

```diff
@@ -22,7 +22,21 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  return names.find(function(name) {
+    var words = name.split(' ');
+    const l = words.length;
```

Comment:
do not use one letter names it is hard to understand, not readable, not searchable